### DISCUSSION TITLE
Reset Frigate+ training images?

### DISCUSSION THREAD

**Discussion by ibrewster** [blakeblackshear/frigate#22668] [Ask A Question]
So, apparently I'm really bad at annotating images. Take the one I've attached, for example. I've drawn the annotation box tight around the car. But the instructions say that if an object is partially obscured/out of frame, you should draw the bounding box to include the missing parts - which in this case means the box should be somewhat taller, although I couldn't tell you *how much* taller (maybe for some of you it's easy, but I'm looking at it and have no idea). To make matters worse, however, in looking through the images I've submitted, some of them *do* have the bounding box taller - I'm thinking those were suggestions that I simply didn't modify.

I can't help but think this is confusing to the model, and have to wonder how many other of the almost 900 images I have submitted are similarly low-quality and making matters worse, not better. As such, given the new YOLO model now available for the google coral edge, I'm thinking I'd like to ditch all my training images and start over, hopefully at least being consistent in how I annotate them. Is this possible and advisable? Or is there a better option?

<img width="685" height="910" alt="Screenshot 2026-03-27 at 3 25 20 PM" src="https://github.com/user-attachments/assets/cbbd29cd-5c97-4e59-ac04-186bbda49a4f" />

**Comment by inkeep-ai-assistant** [comment]
I don't have information about a way to bulk delete or reset all your training images in Frigate+. The documentation doesn't cover account-level image management or deletion features.

However, regarding your annotation concerns:

For objects partly out of frame, the guidance is to label them "only when a person would reasonably be able to recognize the object from the visible parts."[(1)](https://docs.frigate.video/plus/annotating) In your screenshot, the car is clearly recognizable, so labeling it is appropriate. The bounding box should include the full object even when occluded—so if part of the car extends beyond the visible frame, you'd extend the box to where you estimate the full car would be.[(1)](https://docs.frigate.video/plus/annotating)

A few points that may help:

- **AI suggested labels** appear as light blue boxes when annotating, and these are converted to labels when saving. You should remove errant suggestions and keep the most accurate bounding box, deleting duplicates so you have just one label per object.[(1)](https://docs.frigate.video/plus/annotating)

- **Consistency matters more than perfection.** The model training benefits from diverse images across different conditions (cloudy days, sunny days, dawn, dusk, night) with at least 100 verified images per camera recommended.[(2)](https://docs.frigate.video/plus)

- **You can improve going forward** by being more careful with new submissions. The base models are trained on images from all Frigate+ users, so your fine-tuned model builds on that foundation.[(2)](https://docs.frigate.video/plus)

For specific account management questions like deleting submitted images, you'd need to check the Frigate+ web interface directly or reach out through the Frigate community channels.

📚 **Sources:**
1. [Annotating your images](https://docs.frigate.video/plus/annotating)
2. [Models](https://docs.frigate.video/plus)

---

Have another question? Just tag [@inkeep](https://inkeep.com).

**Comment by bbrodriguez** [comment]
I have hundreds of images tagged just like that and it seems fine. Maybe average it out from here on in?

Though I will say I’d like to wipe my face labels and start those again.

**Comment by blakeblackshear** [comment]
There is nothing wrong with car annotations like this. There are thousands of cars that are partially in frame in the training data just like this. No need to reset anything before switching to YOLOv9. All the training data you have now is perfectly fine for all model types. Your submitted images don't have any link to a specific model.

**Comment by ibrewster** [comment]
Ok, that makes sense, but for two things:

1) I have the annotation tight around the visible car in this one. However, in others, I tried to do what the instructions stated, that is "The bounding box should include the full object even when occluded—so if part of the car extends beyond the visible frame, you'd extend the box to where you estimate the full car would be" to quote the AI response. Point being, I have *NOT* been consistent, and trying to go back and re-annotate all the images in the same way would be prohibitively difficult.

2) Perhaps more to the point, I tried the base YOLOv9 model. It was good, but I thought it could be better (low confidence on some people in my living room, for example), so I asked for a training. The fine tuned model was noticeably worse - giving me lots of false positives where before I only had the occasional. To me, this indicates that my annotations have been...bad. 

Which, honestly, makes sense, given that many of the images were submitted back when I first started with frigate, before I had a clue what I was doing... :)

**Comment by blakeblackshear** [comment]
Send me an email at blake@frigate.video and I can take a look.
